platform/security/idp/security-idp-server/src/main/grunt/tasks/bower.js

```diff
@@ -0,0 +1,33 @@
+module.exports = function(grunt) {
```

Comment:
Do we have a ticket to stop repeating these files every time we need to configure grunt/bower? We keep repeating the exact same `bower.js` file everywhere. I assume there's a better way to do this.